content/en/blog/_posts/2023-10-19-Gateway-API-GA.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: blog
+title: "Gateway API is now Generally Available (GA)"
+date: 2023-10-19T10:00:00-08:00
+slug: gateway-api-ga
+---
+
+***Authors:*** Shane Utt (Kong), Nick Young (Isovalent), Rob Scott (Google)
+
+We (the community developing [Gateway API][gwapi]) are incredibly pleased and
+proud to announce _general availability_ of Gateway API for use with Kubernetes
+clusters `v1.24` and newer!
+
+## Important Highlights
+
+Gateway API brings a lot of new networking possibilities to the table in your
+Kubernetes cluster. In this post we'll cover some of the most important
+highlights.
+
+For a complete list of changes for this specific release, please see our
+[v1.0.0 changelog entry][chlogv1].
+
+For a complete list of every feature for every release, or to review some of
+the releases leading up to `v1.0.0`, please see our [historical
+changelog][clog].
+
+### Version 1 APIs
+
+This marks the **first stable release** of Gateway API, where we are now
+publishing the following APIs as `v1` (Version 1):
+
+- [GatewayClass][gwc]
+- [Gateway][gw]
+- [HTTPRoute][hr]
+
+Reaching version 1 with these APIs means we future iterations on them will be
+handled in a backwards compatible manner.
+
+### CEL Migration
+
+Gateway API [CRDs][crds] now include [CEL][cel] validation. This streamlines
+the development of validation for our resources, and reduces the overhead of
+deploying Gateway API to a cluster as it eliminates the need for our historical
+[admission webhook][admw].
+
+> **Note**: For Kubernetes `v1.25` and newer we recommend against deploying the
+> webhook, and if you had it running previously you can now uninstall it in
+> favor of the validation now provided via CEL.
+
+### Kubernetes Support
+
+Gateway API generally [supports the last 5 stable Kubernetes versions][kvs]. As
+such, this release is supported on Kubernetes clusters version `1.24` through
+`1.29` at the time of release.
+
+> **Warning**: Related to our migration to [CEL][cel] as mentioned above, we
+> don't recommend deploying on `v1.24` clusters if it can be avoided as these
+> clusters will still require the admission webhook, and that webhook is
+> considered deprecated and support for it will drop shortly after this GA
+> release when Kubernetes `v1.30` becomes available. If it's unavoidable to
+> deploy on `v1.24`, just note that you'll need to upgrade fairly soon in
+> order to be considered supported.
+
+### Getting Started
+
+If you want to try Gateway API features out for yourself but don't know where
+to get started you'll want to check out one of the 20+ implementations which
+we list on our website:
+
+https://gateway-api.sigs.k8s.io/implementations/
```

Comment:
```suggestion
If you want to try Gateway API features out for yourself but don't know where
to get started you'll want to check out one of the 20+
[implementations](https://gateway-api.sigs.k8s.io/implementations/) which
we list on the Gateway API website.
```

Any plans to add something to CNCF Landscape (or similar)?

Reply:
No plans for that currently in fact, maybe it makes sense to do so after GA
/cc @robscott @youngnick